### ISSUE TITLE
Team 159 Review

### ISSUE THREAD

**Issue by sinansevgi** [Misori-simon/myportfolio#4]
Congratulations it is a great portfolio.

There are some issues: 

**General Issues**
The live demo link for your fembe project doesn't work.
The Reddit profile link in the footer opens the Reddit home page.

**Accessibility issues**
The contrast on your footer doesn't meet WCAG standards (Background color is too light it makes reading text harder.)
You can check [this](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast) guide
